.github/workflows/scripts/dist-tarball-check

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
```

Comment:
This file should have a .sh ending.

Reply:
I don't like putting suffixes on executables in case the script is rewritten in another language.